pkgs/development/tools/rust/cargo-binutils/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform, runCommand }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-binutils";
+  version = "0.3.3";
+
+  src = let
+    repo = fetchFromGitHub {
+      owner = "rust-embedded";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "sha256-Dgn+f4aSsDSh+RC8yvt3ydkdtwib5jEVsnZkod5c7Vo=";
+    };
+  in runCommand "source" { } ''
+    cp -R ${repo} $out
```

Comment:
Can you try to contact upstream regarding them not committing `Cargo.lock`?

Reply:
I've opened a [PR](https://github.com/rust-embedded/cargo-binutils/pull/99) :+1:

Reply:
One of the maintainers of cargo-binutils [said](https://github.com/rust-embedded/cargo-binutils/pull/99#issuecomment-757503230) that they won't push the `Cargo.lock` file

Reply:
They ~can~ should use Nix :).